### PR TITLE
native: update action sheets for consistency

### PR DESCRIPTION
**Pull request by dnbrwstr**
This PR updates all action sheets for consistency with design + adds fixture previews.

Retry send sheet:

Profile sheet (group admin + default versions)

Group preview sheet (various states)

Group join request sheet

Edit section name sheet

Delete confirmation sheet

Create channel sheet

Channel sort sheet

Add gallery post sheet

Attachment sheet

### CODE REVIEW

**Comment by jamesacklin** [comment]
Great, I see we have a form element now as well.

File: packages/ui/src/components/TrimmedText.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        {
+          $s: {
+            '$platform-ios': trimStyle(0, -2),
+            '$platform-android': trimStyle(0, -3),
+          },
+          $m: {
+            '$platform-ios': trimStyle(-5, -5),
+            '$platform-android': trimStyle(-5, -4),
+          },
+          $l: {
```
Comment (latter-bolden) [inline packages/ui/src/components/TrimmedText.tsx:118]: I'm surprised the trim styles are the same here up through _3xl_

**Review by latter-bolden — APPROVED** [review]
Looking good. It'll be easier to have ergonomic feedback on the new base components once actually working with them, but nothing stands out. Have a few spots in mind to start transitioning once these are in.

Nice work on adding all the fixtures too, valuable!

File: packages/ui/src/components/TrimmedText.tsx (hunk 1 above)
Comment (dnbrwstr) [reply packages/ui/src/components/TrimmedText.tsx:118]: Ya, mostly because line height barely changes haha.

**Pull request merged**